modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/portlet/action/UpdateKBArticleMVCActionCommand.java

```diff
@@ -92,6 +92,12 @@ protected void doTransactionalCommand(
 			kbArticle = _kbArticleService.revertKBArticle(
 				resourcePrimKey, version, serviceContext);
 		}
+		else if (cmd.equals(Constants.CANCEL)) {
+			_kbArticleService.unlock(resourcePrimKey);
+
+			hideDefaultSuccessMessage(actionRequest);
+			sendRedirect(actionRequest, actionResponse);
```

Comment:
Opt: not sure about this, but shouldn't there be a blank line between the `hide*` and the `sendRedirect`?

Reply:
The SF test is not complaining about it 🤨